.eslintrc.js

```diff
@@ -0,0 +1,453 @@
+const OFF = 0;
+const WARN = 1;
+const ERROR = 2;
+
+module.exports = {
```

Comment:
How about using the new [@kiwicom/eslint-config](https://www.npmjs.com/package/@kiwicom/eslint-config) as base?

Reply:
Great, It is finally deployed. Thanks, I will do that